terraform/modules/monitoring/metrics/locals.tf

```
@@ -0,0 +1,7 @@
+locals {
+  env = [
+    "Serilog__MinimumLevel=${var.log_level}",
+    "ASPNETCORE_ENVIRONMENT=${var.dev_env}"
+  ]
+  db_env = [for t in keys(var.database_env_vars) : format("%s=%s", t, lookup(var.database_env_vars, t))]
```

This can be simply rewritten like this:
```suggestion
  db_env = [for k, v in var.database_env_vars) : "${k}=${v}"]
```